pkg/metadata/main.go

```diff
@@ -103,6 +103,8 @@ func main() {
 			netProviders = append(netProviders, NewDigitalOcean())
 		case p == "cdrom":
 			cdromProviders = ListCDROMs()
+		case p == "vmware":
```

Comment:
I think cdrom should also be last in this switch-case. Otherwise if one has both 'vmware' and 'cdrom' as a provider it will use cdrom, not vmware provider.

Reply:
You are right cdrom being the last provider makes more sense, the latest commit fixes this